### DISCUSSION TITLE
有關於 Nest.js 部署至平台問題

### DISCUSSION THREAD

**Discussion by paperhsiaooo** [zeabur/.github#27] [General]
我想將後端部署至 zeabur 上，但發現在執行指令 `yarn build` 之後馬上就遇到錯誤，發現都是 @prisma/client 沒有 exported 之類的錯誤
請問是在使用 prisma 在 build 之前有要做什麼步驟嗎？

<img width="1088" alt="image" src="https://github.com/zeabur/.github/assets/127862412/d30b1d6d-97e2-405a-822a-7a6c2beb15f8">

**Comment by yuaanlin** [comment]
您好，若需要技術支援，請您至 Zeabur 官方 Discord 頻道 https://discord.zeabur.com 發文，我們會第一時間回覆您，謝謝！

**Reply by paperhsiaooo** [reply]
連結似乎是 invite invalid

<img width="505" alt="image" src="https://github.com/zeabur/.github/assets/127862412/6a60187b-b204-4487-a8cd-100999ac17d5">

**Reply by yuaanlin** [reply]
https://discord.gg/Au3VupPBWW

**Reply by paperhsiaooo** [reply]
依然如此
<img width="500" alt="image" src="https://github.com/zeabur/.github/assets/127862412/28ff4973-df88-414b-86a8-4d77ffb67e02">

儘管發送認證信也沒在信箱收到（垃圾信件也確認過）
<img width="500" alt="image" src="https://github.com/zeabur/.github/assets/127862412/ef974860-3d0d-4729-920f-3f23af692761">

**Reply by paperhsiaooo** [reply]
@yuaanlin 
 感謝，成功了
